gwakeonlan/settings.py

```diff
@@ -43,6 +44,12 @@ def __init__(self):
     parser.add_option('-q', '--quiet', dest='verbose_level',
                       action='store_const', const=VERBOSE_LEVEL_QUIET,
                       help='hide error and information messages')
+    parser.add_option('-i', '--import-ethers-file', dest='ethers_file',
+                      help='Include the given ethers file')
```

Comment:
I'm not entirely sure I'm willing to add command line arguments for import a bunch of hosts from an external file.
The actual command line doesn't include a way to add or remove a single host, neither to import them from the ARP cache.

A command line argument is meant for non interactive actions while this app is entirely interactive.